website/addons/wiki/templates/edit.mako

```diff
@@ -39,15 +51,9 @@
            </div>
         </div>
 
-        <!-- Menu without toggle in XS size only -->
-        <div class="osf-panel visible-xs">
-            <div class="osf-panel-header"> <i class="fa fa-list"> </i>  Menu </div>
-            <div class="osf-panel-body ">
-                <%include file="wiki/templates/toc.mako"/>
-            </div>
-        </div>
```

Comment:
Yes! The treebeard menu remains visible on small screens. And looks like this: 

![screen shot 2015-06-15 at 10 44 01 am](https://cloud.githubusercontent.com/assets/7913604/8162493/9591eeae-134b-11e5-828f-e03984f1d8dc.png)